src/app/src/test/kotlin/org/imperial/mrc/hint/integration/IntegrationTests.kt

```diff
@@ -76,6 +76,35 @@ abstract class SecureIntegrationTests : CleanDatabaseTests()
                 getTestEntity("survey.csv"))
     }
 
+    fun assertSecureWithSuccess(isAuthorized: IsAuthorized,
```

Comment:
Since this method is checking for a `HttpSstatus` other than 200/success, this name isn't quite right - how about `assertSecureWithHttpStatus`?